changelog.md

```diff
@@ -1,5 +1,9 @@
 # 7TV Web Extension - Changelog
 
+### Version 2.1.2
+
+- Clicking a twitch emote now opens the emote card.
+
```

Comment:
Should this be under version 2.1.2 or 2.1.1?

Reply:
2.1.2, since 2.1.1 was released, although only on Firefox